dom/src/makeDOMDriver.ts

```diff
@@ -1,13 +1,11 @@
 import {Driver, FantasyObservable} from '@cycle/run';
-import {init} from 'snabbdom';
-import {Module} from 'snabbdom/modules/module';
+import {init, Module} from 'snabbdom';
 import xs, {Stream, Listener} from 'xstream';
 import concat from 'xstream/extra/concat';
 import sampleCombine from 'xstream/extra/sampleCombine';
 import {DOMSource} from './DOMSource';
 import {MainDOMSource} from './MainDOMSource';
-import {VNode} from 'snabbdom/vnode';
-import {toVNode} from 'snabbdom/tovnode';
+import {VNode,toVNode} from 'snabbdom';
```

Comment:
Could you move all the imports into a single statement?

Reply:
Merged into line 2! Thanks!